docs/docs/active-learning/in-encord.mdx

```diff
@@ -0,0 +1,105 @@
+---
+sidebar_position: 2
+title: In Encord
+---
+
+import TOCInline from "@theme/TOCInline";
+
+# Active Learning in Encord
+
+**Learn how to set up the components of your active learning process in Encord**
+
+:::note Requirements
+
+Active learning workflows in the Encord platform are specifically designed for workflow projects within the platform.
+This requirement allows for seamless task movement between essential stages such as `label`, `review`, and `complete` when utilizing the SDK.
+
+Note that deviating from this requirement may require customizing certain steps to align with your specific needs.
+
+:::
+
+Active learning workflows in Encord Active share the following key stages:
+<TOCInline toc={toc.filter((node) => node.level === 2).map((node, index) => ({ ...node, value: `${index + 1}. ${node.value}` }))} />
+
+If you prefer to witness an active learning workflow in action please take a look at our end-to-end tutorial about [easy active learning on MNIST][easy-active-learning-in-mnist-tutorial].
+
+## Initialization
+To start an active learning workflow, you need an initial labeled dataset for training the machine learning model.
+In the Encord platform, this corresponds to having a project with annotations.
+
+If you don't have any projects yet, you can watch the tutorial video on setting up a [workflow project][workflow-project-tutorial] to get started quickly.
+
+### Choose an Encord project
+
+To proceed, you should pull the project into Encord Active.
+Execute the following CLI command and remember to acknowledge that you would like to include uninitialized label rows, as they represent unannotated data.
+```bash
+encord-active import project
+```
+If you require detailed information on the options available during the import process, you can refer to the [Import from Encord platform][import-encord-project] guide.  
+
+If your workflow project already contains annotations, you can proceed directly to [Model training and update](#model-training).
+
+[//]: # (Alternatively, if you have a trained model that you intend to use in the active learning workflow, you can jump to the [Querying]&#40;#querying&#41; step.)
+
+## Prioritize high value data to label
+
+If your project does not have any annotations or you are seeking the most appropriate data for labeling, it's essential to score and rank your data.
+While random selection is a possibility, Encord Active provides metrics such as [`Image Diversity`][image-diversity-metric] to enhance and optimize annotation impact.
+This metric ranks images based on their ease of annotation, enabling prioritization of suitable and manageable data.
```

Comment:
Ah, Gorkem rephrased the `Image Difficulty` metric to `Image Diversity` as it can be understand as a diversity-sampling acquisition function. The link doesn't work because I didn't update this branch after that change arrived to main. Will do now.